src/js/quote.form.js

```diff
@@ -0,0 +1,39 @@
+$('form[name="form-quote"]').validate({
+  // required fields
+  nameQuote: 'required',
+  phoneQuote: 'required',
+  projectTypeQuote: 'required',
+  linkedinCareers: 'required',
+  projectNameQuote: 'required',
+  budgetQuote: 'required',
+  detailsQuote: 'required',
+  emailQuote: {
+    required: true,
+    // Specify that email should be validated
+    // by the built-in "email" rule
+    email: true
+  },
+  messages: {
+    emailQuote: "Por favor introduce un correo valido."
+  },
+  submitHandler: function(form) {
+    var data = $('form[name="form-quote"]').serialize();
+    console.log(data);
```

Comment:
Remove console.log and if you dont use the AJAX code remove it

Reply:
removed